templates/org/aac/aac-architecture-guide-content.md

```diff
@@ -17,39 +17,43 @@ categories:
 ---
 
 # H1 title
-
-Other than the H2s listed below, there are no further content requirements for Architecture Guides, except to follow all Microsoft and Learn style and pull request criteria. Diagrams are not required for this content type, but if a diagram is included, it should have a Dataflow/Workflow section underneath the diagram to explain it. See the [Example workload template](sample-solution-templates) sections on Architecture and Dataflow for details.
-
-The Architecture Guide template requires the following sections at the end of the article:
+<!--Other than the H2s listed below, there are no further content requirements for Architecture Guides, except to follow all Microsoft and Learn style and pull request criteria. Diagrams are not required for this content type, but if a diagram is included, it include a subsequent *Dataflow* or *Workflow* section to explain it. For more information, see the [Example workload template](sample-solution-templates). The Architecture Guide template requires the following sections at the end of the article.-->
   
 ## Contributors
+<!--This section is expected, but is optional if all the contributors prefer not to include it.
+
+1. Start with the explanation text in italics that's the same for every article. This makes it clear that Microsoft takes responsibility for the article and not a single contributor.
+
+1. Include the "Principal authors" list and the "Additional contributors" list, if there are additional contributors. This should all be in plain text, not italics or bold.
 
-> (Expected, but this section is optional if all the contributors would prefer to not include it)
+1. Link each contributor's name to the person's LinkedIn profile. 
 
-> Start with the explanation text (same for every article), in italics. This makes it clear that Microsoft takes responsibility for the article (not the one contributor). Then include the "Pricipal authors" list and the "Additional contributors" list (if there are additional contributors) (all in plain text, not italics or bold). Link each contributor's name to the person's LinkedIn profile. After the name, place a pipe symbol ("|") with spaces, and then enter the person's title. We don't include the person's company, MVP status, or links to additional profiles (to minimize edits/updates). (The profiles can be linked to from the person's LinkedIn page, and we hope to automate that on the platform in the future). 
-> Implement this format:
+1. Insert a pipe symbol ("|") with spaces after the name, and then enter the person's title. To minimize edits or updates, don't include the person's company, MVP status, or links to additional profiles. You can link profiles from the person's LinkedIn page. In the future, we aim to automate this process on the platform.
```

Comment:
```suggestion
1. Insert a pipe symbol ("|") with spaces after the name, and then enter the person's title. To minimize edits or updates, don't include the person's company, MVP status, or links to additional profiles. You can link to profiles from the person's LinkedIn page. 
```
Here I recommend deleting "In the future, we aim to automate this process on the platform."